server/views/pages/allocate-to-practitioner.njk

```diff
@@ -29,22 +31,56 @@
 {% endblock %}
 
 {% block content %}
-
-    {{ pop({name: name, crn: crn, tier: tier}) }}
-
-    <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
-
-    <h2 class="govuk-heading-l">You're allocating this case to probation practitioner {{ offenderManagerDetails }}</h2>
-    {% if data.capacity == data.potentialCapacity %}
-        <p id="impact-statement">Their workload will remain at <span class="{{ 'percentage-extra-over' if data.capacity > 100 }}">{{ data.capacity | round(1)}}%</span> as they are already managing this case.</p>
-    {% else %}
-        <p id="impact-statement">This will increase their workload from <span class="{{ 'percentage-extra-over' if data.capacity > 100 }}">{{ data.capacity | round(1)}}%</span> to <span class="{{ 'percentage-extra-over' if data.potentialCapacity > 100 }}">{{ data.potentialCapacity | round(1) }}%</span>.</p>
-    {% endif %}
-    <div class="govuk-form-group govuk-!-margin-top-9">
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
```

Comment:
There's already a `<main />` wrapper in the main layout template so you're needlessly doubling up on those:
<img width="596" alt="image" src="https://github.com/user-attachments/assets/56a50a8e-1e13-4c2a-9e6a-f7eda645c31e">

Also, more importantly, this tag isn't closed which is causing problems in the rendering

Reply:
fixed